src/Juvix/Compiler/Core/Extra/Recursors/Fold/Named.hs

```diff
@@ -0,0 +1,41 @@
+module Juvix.Compiler.Core.Extra.Recursors.Fold.Named where
+
+import Data.Functor.Identity
+import Juvix.Compiler.Core.Extra.Recursors.Base
+import Juvix.Compiler.Core.Extra.Recursors.Fold
+
+ufoldA :: Applicative f => (a -> [a] -> a) -> (Node -> f a) -> Node -> f a
+ufoldA uplus f = ufoldG unitCollector uplus (const f)
+
+ufoldAB :: Applicative f => (a -> [a] -> a) -> (BinderList Info -> Node -> f a) -> Node -> f a
```

Comment:
Should be ufoldLA to make it consistent with the convention for maps. Analogously with all other names in this module.